crates/ursa-pod/benches/e2e.rs

```diff
@@ -114,55 +118,76 @@ fn protocol_benchmarks(c: &mut Criterion) {
                 http_hyper::server_loop,
             );
         }
+
+        #[cfg(feature = "bench-quic")]
+        {
+            let mut g = c.benchmark_group(format!("QUINN UFDP/{range}"));
```

Comment:
minor nit, could you add `QUIC` so that both implementations can be run at once?